### DISCUSSION TITLE
Sumatra PDF viewer doesn't do animations!

### DISCUSSION THREAD

**Discussion by hansworzt** [sumatrapdfreader/sumatrapdf#4620] [Ideas]
I generate PDF files from MikTeX. Unfortunately, Sumatra PDF doesn't do animations - as all the other PDF viewer alternatives I tried don't either, only the Adobe Reader displays them. I would switch from the Adobe one if this feature were present.

**Comment by GitHubRulesOK** [comment]
Animations from LaTeX were always an issue as primarily designed for Adobe (mainly Windows) software so dont run on most other systems.

For 3D files I use Tracker EDITOR(view) it will also run Windows Audio/Video and thus most rich media except the ill fated ShockingWavesFlashy Exploits.
![image](https://github.com/user-attachments/assets/5521ae0a-41ca-4a34-9e97-b310f7c7335e)

Menu or press `T`
![image](https://github.com/user-attachments/assets/b29d713d-a639-4eb8-b2ac-15e259238a25)

![image](https://github.com/user-attachments/assets/a0981c85-328b-4540-aeac-d58ae018ed42)
![image](https://github.com/user-attachments/assets/e6d1191d-2fe2-40be-89d3-f4163bf26f3b)
